### PR TITLE
Hidden hack to use own consumer key/secret

### PR DESCRIPTION
**Pull request by CODeRUS**

### CODE REVIEW

**Comment by siteshwar** [comment]
-1

This pull request might have a very specific use case, but feature requested here looks unnecessary to me.

**Comment by CODeRUS** [comment]
because of that using of this feature is hidden :)

i'd like to have hidden feature than recompiling every update ;)

File: qml/tweetian-harmattan/SignInPage.qml [hunk 1]
```diff
@@ -82,10 +82,40 @@ Click the button below will launch an external web browser for you to sign in to
...
+                        internal.signInButtonClicked()
+                    }
+                    onPressAndHold: {
+                        consKey.visible = true
+                        consSec.visible = true
+                    }
                 }
             }
 
+            TextField {
```
Comment (siteshwar) [inline qml/tweetian-harmattan/SignInPage.qml:105]: You would not want to enter these values every time you start tweetian, you should save them in the settings.

Comment (CODeRUS) [inline qml/tweetian-harmattan/SignInPage.qml:105]: oh damn me, thank you :)

Comment (CODeRUS) [inline qml/tweetian-harmattan/SignInPage.qml:105]: Updated commit :)

**Pull request closed without merging**